File: exs/mundo_1/ruby/018.rb
=begin
Desafio 018

Problema: Faça um programa que leia um ângulo qualquer e mostre na
          tela o valor do seno, cosseno e tangente desse ângulo.

Resolução do problema:
=end

print"Digite o valor do ângulo: "
a = gets.chomp.to_f

seno = Math.sin(a)
cosseno = Math.cos(a)
tangente = Math.cos(a)

puts"O seno do ângulo #{a} é #{seno}."
puts"O cosseno do ângulo #{a} é #{cosseno}."
puts"A tangente do ângulo #{a} é #{tangente}."

File: exs/mundo_1/golang/018.go
/*
Desafio 018

Problema: Faça um programa que leia um ângulo qualquer e mostre na
          tela o valor do seno, cosseno e tangente desse ângulo.

Resolução do problema:
*/

package main

import(
	"fmt"
	"math"
)

var ang float64

func main(){
	fmt.Print("Digite o valor do ângulo: ")
	fmt.Scanln(&ang)

	cos := math.Cosh(ang)
	sin := math.Sin(ang)
	tan := math.Tan(ang)

	fmt.Printf("O seno do ângulo de %.2f graus é igual a %.2f.\n", ang, sin)
	fmt.Printf("O cosseno do ângulo de %.2f graus é igual a %.2f.\n", ang, cos)
	fmt.Printf("A tangente do ângulo de %.2f graus é igual a %.2f.\n", ang, tan)

}
